packages/cli/README.md

```diff
@@ -1681,7 +1681,7 @@ Output Fields are optional, but can be used to:
 - Make sure that custom fields that may not be found in every live sample and - since they're custom to the connected account - cannot be defined in the static sample, can still be mapped.
 - (Added in v15.6.0) Define what field(s) can be used to uniquely identify and [deduplicate](#dedupe) items returned by a polling trigger call.
```

Comment:
Where does `#dedupe` link to? I can't find that anchor.

Reply:
It's defined at this [line](https://github.com/zapier/zapier-platform/blob/75c933e07a9fabf3bda0bc818ad17055b500bdb5/packages/cli/README-source.md?plain=1#L2059) but it doesn't seem to work. Maybe it's should be `<a name="dedupe">` instead of `<a id="dedupe">`. Anyway, I've replaced `#dedupe` with `#how-does-deduplication-work` in [8971473](https://github.com/zapier/zapier-platform/pull/763/commits/897147320e9c3e2db756a945397ffedbe1d1f0a8). Thanks!